packages/blob-reader/helper.js

```diff
@@ -86,6 +86,13 @@ function getPath(el) {
     ret = $('.js-permalink-shortcut')?.getAttribute('href');
   }
 
+  // When current page is an actions run workflow file
+  if (!ret) {
+    ret = $(
+      '.actions-fullwidth-module .text-small .Link--secondary',
```

Comment:
Maybe we should switch to https://github.com/refined-github/github-url-detection 

Jumpcat.dev is using `github-url-detection` and it works really well.

Reply:
How would that help get the url from this link? The value of it is `https://github.com/OctoLinker/OctoLinker/blob/e9e60b630a37e980e48a816e4aaf6355a3fd8707/.github/workflows/ci.yml` and it's the only spot on the page with it.

<img alt="image" src="https://user-images.githubusercontent.com/831974/201791244-155c3e72-f147-435a-b2da-64b16a37d508.png">

Reply:
I shouldn't comment on PRs after midnight 🙈